channeldb/payments.go

```diff
@@ -22,6 +24,19 @@ var (
 	// the status of the payment.
 	paymentStatusKey = []byte("payment-status-key")
 
+	// paymentCreationInfoKey is a key used in the payment's sub-bucket to
+	// store the creation info of the payment.
+	paymentCreationInfoKey = []byte("payment-creation-info")
+
+	// paymentAttemptInfoKey is a key used in the payment's sub-bucket to
+	// store the info about the latest attempt that was done for the
+	// payment in question..
+	paymentAttemptInfoKey = []byte("payment-attempt-info")
+
+	// paymentSettleInfoKey is a key used in the payment's sub-bucket to
+	// store the settle info of the payment.
+	paymentSettleInfoKey = []byte("payment-settle-info")
```

Comment:
Can this just be stored as the final attempt?

Reply:
This stores the preimage, which is not part of the attempt information.